### DISCUSSION TITLE
Error handling for unique constraint on model Column

### DISCUSSION THREAD

**Discussion by samearcher** [jowilf/starlette-admin#417] [Q&A]
Hi, at the moment one of my columns has a unique constraint. Can error handling please be added to elegantly deal with a user entering a duplicate. Not sure if this is already included somewhere since presumably user emails cannot be duplicate?

**Comment by jowilf** [comment]
You can find an example here-> https://github.com/jowilf/starlette-admin/blob/fb62f1620429d50595e4b452ec60ab78b251863a/examples/sqla/views.py#L66.
